src/i18n.js

```diff
@@ -74,6 +75,9 @@ i18n
       ja: {
         translations: translationJa,
       },
+      "pt-BR": {
+        translations: translationPtBR
+      },
```

Comment:
As we don't have any other Portuguese translations, could be possibly to fall back to this region-specific Portuguese when any Portuguese browser is detected?

```suggestion
      },
      "pt": {
        translations: translationPtBR
      },
```